justfile

```diff
@@ -1,5 +1,6 @@
 # We import environment variables from .env
 set dotenv-load := true
+set shell := ["bash", "-uc"]
```

Comment:
Why we set this? If we set this do we still have to specify `#!/usr/bin/env sh` in other jobs?

Reply:
This is the shell that all commands are run with.  I assume you're referring to this:

```
dart-deps:
    #!/usr/bin/env sh
    ....
```

This is what Just calls a "shebang recipe", and it's always preceded by the shell you want to run it with. 

> Shebang recipe bodies are extracted and run as scripts [...].

So long story short: I set `bash` so I can use `=~` and `[[`, but this does not affect the recipes you're referring to, I think.

Reply:
I was wondering if with this definition all commands are implitly shebang recipe for every task or if we still have to it to avoid to have to put \ at the end of a line to have all the commands run in the same shell and not a new shell per line.

Reply:
No, that is unfortunately not how it works, I think 🤔 But I will double-check.

Reply:
From the docs:

> Recipes without an initial shebang are evaluated and run line-by-line [...]

So yeah, if we'd want to remove the backslashes we'd have to include the shebang for every recipe.